compiler/semcall.nim

```diff
@@ -354,7 +354,8 @@ proc getMsgDiagnostic(c: PContext, flags: TExprFlags, n, f: PNode): string =
 proc resolveOverloads(c: PContext, n, orig: PNode,
                       filter: TSymKinds, flags: TExprFlags,
                       errors: var CandidateErrors,
-                      errorsEnabled: bool): TCandidate =
+                      errorsEnabled: bool,
+                      fastReturn = false): TCandidate =
```

Comment:
Why is this `fastReturn` required?

Reply:
prevent it goes to `localError(c.config, n.info, getMsgDiagnostic(c, flags, n, f))`, ensure not break previouse behavior.